assignments/pr-7index.html

```diff
@@ -68,27 +80,22 @@ <h2>Account Details</h2>
                 <label for="passwordInput">Password</label>
                 <input
                     type="password"
-                    placeholder="password"
+                    placeholder="Password"
                     id="passwordInput"
                     name="password"
                     required
                 />
             </div>
-            <h2>Preferences:Favorite</h2>
+            <h2>Preferences</h2>
             <div>
-                <label for="favcolor">Select your favorite color:</label>
-                <input
-                    type="color"
-                    id="favcolor"
-                    name="favcolor"
-                    value="#ff0000"
-                />
+                <label for="colorInput">Color:</label>
+                <input type="color" id="colorInput" name="favoriteColor" " />
 
                 <label for="checkboxInput">Subscribe to newsletter</label>
-                <input type="checkbox" id="checkbox" name="checkboxInput" />
+                <input type="checkbox" id="checkboxInput" name="subscribe" />
 
-                <label for="preferred-language">Preferred Language:</label>
-                <select id="preferred-language" name="preferred-language">
+                <label for="languageSelect">Preferred Language:</label>
+                <select id="languageSelect" name="languageSelect">
```

Comment:
You do not ask what is your language select, right? The Right one will be what is your preferred language. so, name attr should be preferredLanguage